### PR TITLE
hiccup-for-blocks, show home tab

### PR DESCRIPTION
**Pull request by mhuebert**
Started working on the home tab, and then got frustrated by the block syntax, so added a little hiccup-parser.

### CODE REVIEW

File: minimal-shadow-cljs-nodejs/src/server/main.cljs [hunk 1]
```diff
@@ -74,7 +76,7 @@
 (defn send-slack-blocks+ [blocks channel]
   (p/->> (slack/post-query-string+ "chat.postMessage"
                                    {:channel channel        ;; id or name
-                                    :blocks (clj->json blocks)})
+                                    :blocks (blocks/to-json blocks)})
```
Comment (daveliepmann) [inline minimal-shadow-cljs-nodejs/src/server/main.cljs:79]: Love this interface. Makes for such a clear call site.

File: minimal-shadow-cljs-nodejs/src/server/slack_events.cljs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+(ns server.slack-events
```
Comment (daveliepmann) [inline minimal-shadow-cljs-nodejs/src/server/slack_events.cljs:1]: Could this be bundled with the existing slack ns?

**Comment by mhuebert** [comment]
> previous easy path was to copy/paste a JSON example, run `json->edn` in emacs, and it just worked

that should still be ok - when the hiccup parser encounters a map, it returns the map itself after recursively crawling the map's values looking for nested hiccup. So, without having tested to verify, I think it is an idempotent function and will return existing, non-hiccup forms as-is. This all rests on the assumption that there are no circumstances do we have post-processed forms that look like hiccup, ie vectors starting with a keyword child.

File: minimal-shadow-cljs-nodejs/src/server/slack_events.cljs (hunk 2 above)
Comment (mhuebert) [reply minimal-shadow-cljs-nodejs/src/server/slack_events.cljs:1]: yes - i think it may already be in a later pr

**Pull request merged**